src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs

```diff
@@ -84,20 +87,17 @@ private static void CopyFrameworkToBuildDirectory(string pathToProject, IDiagnos
                 return;
             }
 
-            var packageName = SentryPackageInfo.GetName();
-            var frameworkDirectory = PlayerSettings.iOS.sdkVersion == iOSSdkVersion.DeviceSDK ? "Device" : "Simulator";
-
-            var frameworkPath = Path.GetFullPath(Path.Combine("Packages", packageName, "Plugins", "iOS", frameworkDirectory, "Sentry.framework"));
-            if (Directory.Exists(frameworkPath))
+            if (Directory.Exists(pathToSentryFramework))
             {
-                logger?.LogDebug("Copying Sentry.framework from '{0}' to '{1}'", frameworkPath, targetPath);
+                logger?.LogDebug("Copying 'Sentry.framework' from '{0}' to '{1}'", pathToSentryFramework, targetPath);
 
-                Directory.CreateDirectory(Path.Combine(pathToProject, "Frameworks"));
-                FileUtil.CopyFileOrDirectoryFollowSymlinks(frameworkPath, targetPath);
```

Comment:
is there a reason for the `FollowSymlinks` removal?

Reply:
According to the [docs](https://docs.unity3d.com/ScriptReference/FileUtil.CopyFileOrDirectoryFollowSymlinks.html) FollowSymlinks is `macOS only` and issues keep coming up with the copy-process failing on other platforms than macOS. We're not relying on symlinks so I don't see a reason why we should special case it.

Reply:
thanks :)